zebra-rpc/tests/vectors/mod.rs

```diff
@@ -0,0 +1,1708 @@
+      "outCiphertext": "048b922d6d688e410aa7e60438923be5cc24b530dab5d4bc44a001cfad987e8b91222bda2b41f66dd129e4a8e946107a6673304694cf75d3bb0dcc4feed5aa3f8a6d832dd7ede2d17dba5fed68c4d68d",
+      "proof": "96ac588d69e360f0a9b5f2e02f4948b12a029e6d762d9a9bb93d9a8cceb8c4fe1c0d813436e7fee186057248a826c4c594d39dee0419b27a857862c7c1c1959a4802144f76647094bae84f55dd1f9611c86452671fb1df744ed720b3a5764902195302bd09a42379ef4dec9f6c6d850fd58865b1cbd40c7e7826cdbc1a88a6a35558a026185a781acdffef6757a77fcfa7bbaa72e2a713818223cef3b561159a42a41d6c49cc0fe16b5e1f1c3565dac12bf6a753d764f751790a8e6ebf066db8"
+    }
+  ],
+  "valueBalance": 0.00005,
+  "valueBalanceZat": 5000,
+  "size": 4009,
+  "time": 1694339033
+}
+"#;
+
+pub const GET_BLOCK_TEMPLATE_RESPONSE_TEMPLATE: &str = r#"
+{
+  "capabilities": [
+    "proposal"
+  ],
+  "version": 4,
+  "previousblockhash": "000000000091f6c52c116112e6ab784cd41550b5a0fbca1c84e6ab62f619eb2e",
+  "blockcommitmentshash": "c78325bc39115ce5bfdd75215e0104aacb674e2dc19d06a8fca4f963b52c462c",
+  "lightclientroothash": "c78325bc39115ce5bfdd75215e0104aacb674e2dc19d06a8fca4f963b52c462c",
+  "finalsaplingroothash": "c78325bc39115ce5bfdd75215e0104aacb674e2dc19d06a8fca4f963b52c462c",
+  "defaultroots": {
+    "merkleroot": "c9a6321293a3955b9970750b0fee21e89bc6f8938e51391410c3577ad9d03405",
+    "chainhistoryroot": "e1c60e427c655cf06400239bc0a42532a993995e7601ef8848c53fd6ea426055",
+    "authdataroot": "ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff",
+    "blockcommitmentshash": "c78325bc39115ce5bfdd75215e0104aacb674e2dc19d06a8fca4f963b52c462c"
+  },
+  "transactions": [
+  ],
+  "coinbasetxn": {
+    "data": "0400008085202f89010000000000000000000000000000000000000000000000000000000000000000ffffffff05039bbc2c00ffffffff0240597307000000001976a914ba92ff06081d5ff6542af8d3b2d209d29ba6337c88ac20bcbe000000000017a914c20cd5bdf7964ca61764db66bc2531b1792a084d87000000009bbc2c000000000000000000000000",
+    "hash": "c9a6321293a3955b9970750b0fee21e89bc6f8938e51391410c3577ad9d03405",
+    "authdigest": "ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff",
+    "depends": [
+    ],
+    "fee": 0,
+    "sigops": 1,
+    "required": true
+  },
+  "longpollid": "0002931866cc3a357e1747853232000000000000000000",
+  "target": "0000000001da6700000000000000000000000000000000000000000000000000",
+  "mintime": 1747847833,
+  "mutable": [
+    "time",
+    "transactions",
+    "prevblock"
+  ],
+  "noncerange": "00000000ffffffff",
+  "sigoplimit": 20000,
+  "sizelimit": 2000000,
+  "curtime": 1747848159,
+  "bits": "1c01da67",
+  "height": 2931867,
+  "maxtime": 1747853232
+}
+"#;
```

Comment:
It would be nice to put these in their own JSON files, maybe in `zebra-test` with the test block vectors.

```suggestion
pub const GET_BLOCKCHAIN_INFO_RESPONSE: &str =
    include_str!("getblockchaininfo_response.json").trim();

pub const GET_BLOCK_RESPONSE_2: &str = include_str!("getblock_response_2.json").trim();

pub const GET_RAW_TRANSACTION_RESPONSE_TRUE: &str =
    include_str!("getrawtransaction_response_true.json").trim();

pub const GET_BLOCK_TEMPLATE_RESPONSE_TEMPLATE: &str =
    include_str!("getblocktemplate_response_template.json").trim();
```

Reply:
Done in 6b141a2ec but I left it here - it won't be compiled into the crate since it's in the integration test folder. But I can move to `zebra-test` if you think that's better.